AppCreationScripts/apps.json

```diff
@@ -33,18 +33,16 @@
             "settingFile": "/app/src/main/res/raw/auth_config_multiple_account.json",
             "replaceTokens": {
                 "appId": "0984a7b6-bc13-4141-8b0d-8f767e136bb7",
-                "tenantId": "common",
                 "redirectUri": "msauth://com.azuresamples.msalandroidapp/1wIqXSqBj7w%2Bh11ZifsnqwgyKrY%3D",
-                "signInAudience": "AzureADandPersonalMicrosoftAccount"
+                "authorityEndpointHost": "https://login.microsoftonline.com/"
              }
           },
           {
             "settingFile": "/app/src/main/res/raw/auth_config_single_account.json",
             "replaceTokens": {
                 "appId": "0984a7b6-bc13-4141-8b0d-8f767e136bb7",
-                "tenantId": "common",
```

Comment:
Since the authorityEndpointHost is just the domain, do you still need the "common" to be replaced with the tenantId?